examples/bzlmod_build_file_generation/MODULE.bazel

```diff
@@ -46,40 +46,81 @@ python = use_extension("@rules_python//python/extensions:python.bzl", "python")
 
 # This name is passed into python.toolchain and it's use_repo statement.
 # We also use the same name for python.host_python_interpreter.
-PYTHON_NAME = "python3"
+PYTHON_NAME_39 = "python_39"
+
+PYTHON_TOOLCHAINS_39 = PYTHON_NAME_39 + "_toolchains"
```

Comment:
I need to move this into the bzlmod example, not here. I doubt gazelle works with multiple toolchains.

Reply:
Yeah, it won't work, so better to keep the `bzlmod` `gazelle` example smaller.

Reply:
I added the code to the examples/bzlmod example and found a bug because I started using submodules.